docs-source/content/userguide/managing-domains/model-in-image/new-dynamicupdate-condition.md

```diff
@@ -0,0 +1,89 @@
+
+```
+
+If the changes involve non-dynamic mbean attributes, and you have specified 'CancelUpdate' under `domain.spec.configuration.model.onlineUpdate.onNonDynamicChanges`, you will see this
+
+```
+Conditions:
+    Last Transition Time:  2021-01-20T15:07:29.427Z
+    Message:               Online update completed successfully, but the changes require restart and the domain resource  specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CancelUpdate'  option to cancel all changes if restart require. The changes are:
+
+Server re-start is REQUIRED for the set of changes in progress.
+
+The following non-dynamic attribute(s) have been changed on MBeans
+that require server re-start:
+MBean Changed : com.bea:Name=oracle.jdbc.fanEnabled,Type=weblogic.j2ee.descriptor.wl.JDBCPropertyBean,Parent=[sample-domain1]/JDBCSystemResources[Bubba-DS],Path=JDBCResource[Bubba-DS]/JDBCDriverParams/Properties/Properties[oracle.jdbc.fanEnabled]
+Attributes changed : Value
+    Reason:                      BackoffLimitExceeded
+    Status:                      True
+    Type:                        Failed
+  Introspect Job Failure Count:  3
+  Message:                       Online update completed successfully, but the changes require restart and the domain resource  specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CancelUpdate'  option to cancel all changes if restart require. The changes are:
+
+Server re-start is REQUIRED for the set of changes in progress.
+
+The following non-dynamic attribute(s) have been changed on MBeans
+that require server re-start:
+MBean Changed : com.bea:Name=oracle.jdbc.fanEnabled,Type=weblogic.j2ee.descriptor.wl.JDBCPropertyBean,Parent=[sample-domain1]/JDBCSystemResources[Bubba-DS],Path=JDBCResource[Bubba-DS]/JDBCDriverParams/Properties/Properties[oracle.jdbc.fanEnabled]
+Attributes changed : Value
+  Reason:    BackoffLimitExceeded
+
+```
+
+If the changes involve non-dynamic mbean attributes, and you have specified 'CommitUpdateOnly' under `domain.spec.configuration.model.onlineUpdate.onNonDynamicChanges` or not set, you will see this               
+      
+```
+  Conditions:
+    Last Transition Time:  2021-01-20T15:09:15.209Z
+    Message:               Online update completed successfully, but the changes require restart and the domain resource specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CommitUpdateOnly' or not set. The changes are committed but the domain require manually restart to  make the changes effective. The changes are: Server re-start is REQUIRED for the set of changes in progress.
+
+The following non-dynamic attribute(s) have been changed on MBeans 
+that require server re-start:
+MBean Changed : com.bea:Name=oracle.jdbc.fanEnabled,Type=weblogic.j2ee.descriptor.wl.JDBCPropertyBean,Parent=[sample-domain1]/JDBCSystemResources[Bubba-DS],Path=JDBCResource[Bubba-DS]/JDBCDriverParams/Properties/Properties[oracle.jdbc.fanEnabled]
```

Comment:
(1) Let's meet sometime to word-smith the 'Message' and 'Reason' text a bit.
(2) Can the message include a list of servers that need restarted? Or maybe instead mention the new label?
(3) Is this condition regenerated if the operator is restarted (does it then detect servers with the restart label).

Reply:
The text is returned from WLS,  that's not much we can do about it.  
The condition and pod labels remains until it is removed, operator restart will not change it.
Currently, the restart server list is internal format from WDT, user probably won't understand and it is intended for future use.

Reply:
OK.